sdk/python/feast/registry.py

```diff
@@ -30,10 +30,21 @@
 
 
 class Registry:
+    """
+    Registry: Used for interfacing with the object store registry.
```

Comment:
Sorry, another nitpick. `Foo is an interface for interacting with a foo` is not really descriptive enough. And calling it an interface is also a little bit misleading.

How about `A registry allows for the management and persistence of feature definitions and related metadata`?